src/Containers/CadView.jsx

```diff
@@ -33,6 +33,7 @@ import {addSceneLayer} from './SceneLayer'
 export const searchIndex = new SearchIndex()
 let count = 0
 
+const useViewRules = window.location.href.match(/(view=CH.SIA380-1.heatmap)/ig)?.length > 0
```

Comment:
Can you move this to IfcViewerApiExtended?  We're trying to limit any changes to CadView, so should avoid any changes and looks like this doesn't actually have a dep.